src/libman/Commands/InstallCommand.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
```

Comment:
OK, I'm blind, which bit of code needs this?

Reply:
Line 115 `Path.Combine()`

Reply:
Yep, there it is. Thanks!

---
In reply to: [195171609](https://github.com/aspnet/LibraryManager/pull/158#discussion_r195171609) [](ancestors = 195171609)